sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -3736,6 +3736,7 @@ public void testTwoTimersSettingEachOtherWithCreateAsInput() {
       ValidatesRunner.class,
       UsesStatefulParDo.class,
       UsesTimersInParDo.class,
+      UsesTestStream.class,
```

Comment:
I had to add this one because it was not correctly categorized and VR were breaking

Reply:
It seems is reported here
https://issues.apache.org/jira/browse/BEAM-9083

Reply:
Shouldn't the VR tests exclude UsesTestStreamWithOutputTimestamp

Reply:
That's the line exactly below that one. Notice that Spark runner does not implement TestStream so we probably need to make the test belong to both categories.

Reply:
Mind to maybe lgtm / merge this one @lukecwik. Seems most VR are broken because of it.

Reply:
I see, I was under the impression that we would place things in the most appropriate category but now thinking about it makes sense to place it in this category as well.